### PR TITLE
Disable Torch2 Dynamo on linux for now as it breaks diffusers

### PR DESCRIPTION
**Pull request by ElliottLester**
## Checklist before requesting a review
- [X] This is based on the /dev branch (Or a fork of it)
- [ ] This was created or at least validated using a proper IDE
- [X] I have tested this code and validated any modified functions
- [ ] I have added the appropriate documentation and hint strings if adding or changing a user-facing feature

### CODE REVIEW

**Comment by butchmarshall** [comment]
This set me on the right track for fixing the issue https://github.com/d8ahazard/sd_dreambooth_extension/issues/1192 

I had to set this at the top of `webui.sh`

**Comment by d8ahazard** [comment]
A more salient fix would probably be to add something like this to preload or install:

# If running linux, disable torch2 dynamo
if os.name == "posix":
    # For now disable Torch2 Dynamo
    os.environ["TORCHDYNAMO_DISABLE"] = 1

**Comment by nbollman** [comment]
Please help me with my vanity...

awww common... wtf am I doing wrong here? I just want to teach Stable Diffusion my face... Ubuntu 22.04 RTX 3090 Everything works, unless I try to train a model in dreambooth. I either have to disable  Gradient Checkpointing when training in dreambooth, and then I run out of VRAM trying to train. But when its enabled I still am getting this error after adding these lines to webui.sh

#!/usr/bin/env bash
#################################################
# Please do not make any changes to this file,  #
# change the variables in webui-user.sh instead #
#################################################

# For now disable Torch2 Dynamo, its screwing up diffusers
if [[ os.name == "posix": ]]; then
    os.environ["TORCHDYNAMO_DISABLE"] = 1
fi


>>ERROR AFTER STARTING TRAINING w >>
Exception training model: ' from user code: File "/home/jon/stable-diffusion-webui/venv/lib/python3.10/site-packages/torch/random.py", line 23, in get_rng_state return default_generator.get_state() Set torch._dynamo.config.verbose=True for more information You can suppress this exception and fall back to eager by setting: torch._dynamo.config.suppress_errors = True '.
 
 The webui.sh changes do not appear to have affected torchdynamo from running...

**Comment by imbaqqqq** [comment]
> Please help me with my vanity...
> 
> python: 3.10.9  •  torch: 2.0.1+cu117  •  xformers: 0.0.17  •  gradio: 3.28.1  •  commit: [5ab7f213]  •  checkpoint: [6ce0161689]
> 
> awww common... wtf am I doing wrong here? I just want to teach Stable Diffusion my face... Ubuntu 22.04 RTX 3090 Everything works, unless I try to train a model in dreambooth. I either have to disable Gradient Checkpointing when training in dreambooth, and then I run out of VRAM trying to train. But when its enabled I still am getting this error after adding these lines to webui.sh
> 
> ```
> #!/usr/bin/env bash
> #################################################
> # Please do not make any changes to this file,  #
> # change the variables in webui-user.sh instead #
> #################################################
> 
> # For now disable Torch2 Dynamo, its screwing up diffusers
> if [[ os.name == "posix": ]]; then
>     os.environ["TORCHDYNAMO_DISABLE"] = 1
> fi
> ```
> 
> > > ERROR AFTER STARTING TRAINING w >>
> > > Exception training model: ' from user code: File "/home/jon/stable-diffusion-webui/venv/lib/python3.10/site-packages/torch/random.py", line 23, in get_rng_state return default_generator.get_state() Set torch._dynamo.config.verbose=True for more information You can suppress this exception and fall back to eager by setting: torch._dynamo.config.suppress_errors = True '.
> 
> The webui.sh changes do not appear to have affected torchdynamo from running...

edit webui.py,like this:
if os.name == "posix":
# For now disable Torch2 Dynamo
os.environ["TORCHDYNAMO_DISABLE"] = "1"

**Comment by ashleykleynhans** [comment]
The fix for this is actually already implemented in `preload.py` on the `dev` branch, so this PR can be closed.
